package.json

```
@@ -13,7 +13,7 @@
     "src"
   ],
   "scripts": {
-    "test": "qunit tests/*-test.js",
+    "test": "qunit tests/*-test.js tests/**/*-test.js",
```

```suggestion
    "test": "qunit  tests/**/*-test.js",
```
This is enough to catch all

---

I tried but it didn't work. It was picking the test files present inside the subdirectories only and not the ones that are directly available under the tests directory